src/interfaces/StudentInterface.ts

```diff
@@ -21,7 +21,7 @@ export enum studentStatus {
 
 export interface StudentInterface {
   id: string;
-  userId: string;
+  //userId: string;
```

Comment:
dlaczego zostało to zakomentowane?

Reply:
Encja studenta nie chciała mi się  na początku zakładać na bazie i dopiero, jak troche zmieniłam sposób zapisu relacji OneToOne,  tabela students założyła się poprawnie, ale interfejs krzyczał wtedy, że nie widzi takiego pola/kolumny jak userId...  Ale jeśli u Was wszystko działa dobrze to oczywiście należy to przywrócić - bo zkomentowaniem tego też sobie mocno utrudniłam życie.